roles/keystone/tasks/keystone_juno.yml

```diff
@@ -0,0 +1,249 @@
+---
+- name: install keystone components
+  yum: pkg={{item}} state=present
+  with_items:
+    - openstack-keystone
+    - openstack-utils
+    - httpd
+  tags: keystone
+
+- name: set keystone configuration
+  command: openstack-config {{ item }}
+  with_items:
+    - " --set /etc/keystone/keystone.conf DEFAULT admin_token {{keystone_admin_token}}"
```

Comment:
This won't be idempotent if `--verbose` and `changed_when` isn't used.  Do we care at this point?  Or v2?